test/unit/es-index-schema.test.js

```diff
@@ -5,6 +5,7 @@ describe('elastic search schema', () => {
   it('returns the schema', () => {
     const properties = Object.keys(schema())
     expect(properties).to.include('carrierType')
-    expect(properties.length).to.equal(88)
+    // At last count, we have 89 bib-level field mappings:
```

Comment:
Is this comment supposed to not match the number being tested?

Reply:
The test as written was brittle because it breaks every time we add a prop. The test now asserts that the number of bib properties exceeds 88, which as of this change it does.